portfolio/src/main/webapp/style.css

```diff
@@ -104,6 +108,7 @@ a {
 
 .card-subheading {
   filter: invert(1);
+  display:none;
```

Comment:
add a space after `:`

Reply:
Rectified in commit here and in other places with 4c79bad